qiskit/tools/events/progressbar.py

```diff
@@ -109,12 +109,26 @@ def finished(self):
 class TextProgressBar(BaseProgressBar):
     """
     A simple text-based progress bar.
+
+    output_handler : where the progress bar should be written to, default
+                     is sys.stdout. Can pass in the output handler itself
+                     or a string such as 'err' to represent the desired
+                     handler.
     """
 
-    def __init__(self):
+    def __init__(self, output_handler=None):
         super().__init__()
         self._init_subscriber()
 
+        self.output_handler = sys.stdout
+
+        if isinstance(output_handler, str):
```

Comment:
Would it make sense to just accept a handler directly as `output_handler`, that might have also the nice side effect of just being able to set `sys.stdout` as the default? This way we would not need to involve a "magic string" in the process and might be clearer for end users.

Reply:
I did think this, but also I felt that allowing the user to enter a string might be more helpful to less experienced python users. Currently, a user can send a handler directly if they wish to, but I can make this the only option if that would be preferred.

Reply:
I would agree that the strings are unnecessary. The docstrings can layout the options, and it is unlikely that an inexperienced Python user would actually need such a feature.

Reply:
That is a good point, I will update it now.